identifiers/service_product.txt

```diff
@@ -277,7 +278,7 @@ MetaDirectory Server
 Metabase
 Metasploit
 MiniDLNA
-MiniUPnP
+MiniUPnPd
```

Comment:
There is actually a `MiniUPnPd` (daemon) and `MiniUPnPc` (client) so I changed this product value.